kitchensink/package-lock.json

```diff
@@ -3234,9 +3234,9 @@
       }
     },
     "classnames": {
-      "version": "2.2.6",
-      "resolved": "https://registry.npmjs.org/classnames/-/classnames-2.2.6.tgz",
-      "integrity": "sha512-JR/iSQOSt+LQIWwrwEzJ9uk0xfN3mTVYMwt1Ir5mUcSN6pU+V4zQFFaJsclJbPuAUQH+yfWef6tm7l1quW3C8Q=="
+      "version": "2.3.1",
```

Comment:
This looks wrong 🤔

Reply:
Could you try and run `npm run clean` and then `npm install` again?

Reply:
will do

Reply:
never mind. like discussed it is used by other libraries